src/indexd/index/drivers/alchemy.py

```diff
@@ -4,26 +4,17 @@
 import uuid
 from contextlib import contextmanager
 
+import sqlalchemy as sa
+from deprecated import deprecated
 from sqlalchemy import (
-    BigInteger,
-    Column,
-    DateTime,
-    ForeignKey,
-    ForeignKeyConstraint,
-    Index,
-    Integer,
-    String,
     and_,
```

Comment:
Functions such as `and_` were removed from imports in other modules and referenced through the sqlalchemy alias "sa". Is there any reason why we didn't do the same here?

Reply:
There's no specific reason, just didn't get to them and there a lot more usage of `and_`